cloudtrail/lib/botocore/data/kms/2014-11-01/examples-1.json

```diff
@@ -0,0 +1,1750 @@
+      }
+    ],
+    "CreateAlias": [
+      {
+        "input": {
+          "AliasName": "alias/ExampleAlias",
+          "TargetKeyId": "1234abcd-12ab-34cd-56ef-1234567890ab"
+        },
+        "comments": {
+          "input": {
+            "AliasName": "The alias to create. Aliases must begin with 'alias/'. Do not use aliases that begin with 'alias/aws' because they are reserved for use by AWS.",
+            "TargetKeyId": "The identifier of the KMS key whose alias you are creating. You can use the key ID or the Amazon Resource Name (ARN) of the KMS key."
+          }
+        },
+        "description": "The following example creates an alias for the specified KMS key.",
+        "id": "to-create-an-alias-1477505685119",
+        "title": "To create an alias"
+      }
+    ],
+    "CreateCustomKeyStore": [
+      {
+        "input": {
+          "CloudHsmClusterId": "cluster-1a23b4cdefg",
+          "CustomKeyStoreName": "ExampleKeyStore",
+          "KeyStorePassword": "kmsPswd",
+          "TrustAnchorCertificate": "<certificate-goes-here>"
+        },
+        "output": {
+          "CustomKeyStoreId": "cks-1234567890abcdef0"
+        },
+        "comments": {
+          "input": {
+            "CloudHsmClusterId": "The ID of the CloudHSM cluster.",
+            "CustomKeyStoreName": "A friendly name for the custom key store.",
+            "KeyStorePassword": "The password for the kmsuser CU account in the specified cluster.",
+            "TrustAnchorCertificate": "The content of the customerCA.crt file that you created when you initialized the cluster."
+          },
+          "output": {
+            "CustomKeyStoreId": "The ID of the new custom key store."
+          }
+        },
+        "description": "This example creates a custom key store that is associated with an AWS CloudHSM cluster.",
+        "id": "to-create-an-aws-cloudhsm-custom-key-store-1628627769469",
+        "title": "To create an AWS CloudHSM custom key store"
+      }
+    ],
+    "CreateGrant": [
+      {
+        "input": {
+          "GranteePrincipal": "arn:aws:iam::111122223333:role/ExampleRole",
+          "KeyId": "arn:aws:kms:us-east-2:444455556666:key/1234abcd-12ab-34cd-56ef-1234567890ab",
+          "Operations": [
+            "Encrypt",
+            "Decrypt"
+          ]
+        },
+        "output": {
+          "GrantId": "0c237476b39f8bc44e45212e08498fbe3151305030726c0590dd8d3e9f3d6a60",
+          "GrantToken": "AQpAM2RhZTk1MGMyNTk2ZmZmMzEyYWVhOWViN2I1MWM4Mzc0MWFiYjc0ZDE1ODkyNGFlNTIzODZhMzgyZjBlNGY3NiKIAgEBAgB4Pa6VDCWW__MSrqnre1HIN0Grt00ViSSuUjhqOC8OT3YAAADfMIHcBgkqhkiG9w0BBwaggc4wgcsCAQAwgcUGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMmqLyBTAegIn9XlK5AgEQgIGXZQjkBcl1dykDdqZBUQ6L1OfUivQy7JVYO2-ZJP7m6f1g8GzV47HX5phdtONAP7K_HQIflcgpkoCqd_fUnE114mSmiagWkbQ5sqAVV3ov-VeqgrvMe5ZFEWLMSluvBAqdjHEdMIkHMlhlj4ENZbzBfo9Wxk8b8SnwP4kc4gGivedzFXo-dwN8fxjjq_ZZ9JFOj2ijIbj5FyogDCN0drOfi8RORSEuCEmPvjFRMFAwcmwFkN2NPp89amA"
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 467d4bab-f23f-4831-975f-4402e58f5ac0 -->